src/Series.cpp

```diff
@@ -3049,17 +3049,30 @@ namespace
         };
     }
 
+    namespace
+    {
+        auto sanitize_regex(std::string const &input) -> std::string
+        {
+            // need to escape special characters reserved for regexes, see
+            // https://stackoverflow.com/questions/40195412/c11-regex-search-for-exact-string-escape
+            std::regex specialChars{R"([-[\]{}()*+?.,\^$|#\s])"};
```

Comment:
Escaping backslashes https://regex101.com/r/GDPK7E/3
```suggestion
            std::regex specialChars{R"([-[\]{}()*+?.,\^$|#\s\\])"};
```